phpBB/includes/functions.php

```diff
@@ -4969,6 +4969,8 @@ function page_header($page_title = '', $display_online_list = false, $item_id =
 		}
 	}
 
+	$user->update_session_infos();
```

Comment:
Well then we still need the call in the footer, where we don't know whether we already did this?